src/Tizen.NUI.Components/Style/NumberPickerStyle.cs

```diff
@@ -0,0 +1,82 @@
+﻿/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.Binding;
+
+namespace Tizen.NUI.Components
+{
+    /// <summary>
+    /// NumberPickerStyle is a class which saves NumberPickerStyle's ux data.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class NumberPickerStyle : ControlStyle
+    {
+        /// <summary>
+        /// Creates a new instance of a NumberPickerStyle.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public NumberPickerStyle() : base()
+        {
+        }
+
+        /// <summary>
+        /// Creates a new instance of a NumberPickerStyle with style.
+        /// </summary>
+        /// <param name="style">Creates NumberPickerStyle by style customized by user.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public NumberPickerStyle(NumberPickerStyle style) : base(style)
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets the NumberPickerStyle Item TextLabel style.
+        /// This style is applied if NumberPickerStyle Item is a TextLabel.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public TextLabelStyle NumberPickerItemTextLabel { get; set; } = new TextLabelStyle();
+
+        /// <summary>
+        /// Gets or sets the NumberPickerStyle Center line style.
+        /// </summary>
+        public ViewStyle NumberPickerCenterLine { get; set;} = new ViewStyle();
```

Comment:
In my opinion, `Divider ` is commonly used to refer the lines between numbers.

Reply:
good idea thanks